.github/PULL_REQUEST_TEMPLATE/simple_pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+<!-- This template should only be used in rare situations. If it is possible that the details will at all effect the ability of other contributors, the detailed pull request template should be used. If your reviewer believes more info is necessary, they may have you complete the detialed_pull_request_template. -->
+
+## Describe your changes
+
+## Issue ticket number and link
+
+## Checklist before requesting a review
+
+- [ ] I have performed a self-review of my code
```

Comment:
I would hope they did but if they didn't I believe it would be obvious. Instead I would encourage people to instead make sure their Pull request is in line with the Code of Conduct and Contributing Guildlines

```suggestion
- [ ] I have reviewed the Code of Conduct
- [ ] I have reviewed the Contributing Guide
```

Reply:
got it makes sense, sounds better this way.